net/tor/src/opnsense/mvc/app/models/OPNsense/Tor/HiddenService.xml

```diff
@@ -0,0 +1,15 @@
+<model>
```

Comment:
does this really need a separate model for the acl?

not a big fan of multiple models per plugin, but it's ok as long as global and item logic are not split in an already multi-model plugin